pipeline/__init__.py

```diff
@@ -2,16 +2,17 @@
     DefaultScheduleStatus,
     Definitions,
     EnvVar,
+    PipesSubprocessClient,
     ScheduleDefinition,
     define_asset_job,
     load_assets_from_modules,
 )
 
-from .assets import gtfs, radvis, sharing, traffic_incidents
+from .assets import gtfs, radvis, sharing, traffic_incidents, webcams
 from .resources import JsonWebAssetIOManager, LamassuResource, PostGISGeoPandasIOManager
 from .resources.gdal import Ogr2OgrResource
 
-assets = load_assets_from_modules([sharing, radvis, gtfs, traffic_incidents])
+assets = load_assets_from_modules([sharing, radvis, gtfs, traffic_incidents, webcams])
```

Comment:
Shall we make this multi-line, in order to have a `git blame`-able formatting from now on?

```suggestion
assets = load_assets_from_modules([
    sharing,
    radvis,
    gtfs,
    traffic_incidents,
    webcams
])
```